src/frontend/src/flows/authenticate/index.ts

```diff
@@ -163,12 +163,12 @@ const pageContent = (
     <div class="textLink">
       <hr />
       <div class="buttonContainer">
-        <button id="recoverButton" class="linkStyle">Lost access?</button>
+        <a id="recoverButton" class="linkStyle">Lost access?</a>
```

Comment:
According to the documentation, the `<a>` tag should have `href` attribute. In this case, there is an action bound to the element. The best solution probably could be to have a proper `href` if only navigation occurs after click or leave the `button` tag.

https://www.w3.org/TR/2016/REC-html51-20161101/textlevel-semantics.html#the-a-element
> If the [a](https://www.w3.org/TR/2016/REC-html51-20161101/textlevel-semantics.html#elementdef-a) element has no [href](https://www.w3.org/TR/2016/REC-html51-20161101/links.html#element-attrdef-a-href) attribute, then the element [represents](https://www.w3.org/TR/2016/REC-html51-20161101/dom.html#represent) a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element’s contents.